core/src/test/scala/unit/kafka/server/ReplicaFetcherThreadTest.scala

```diff
@@ -743,6 +745,52 @@ class ReplicaFetcherThreadTest {
     verify(mockBlockingSend)
   }
 
+  @Test
+  def shouldReassignmentBytesInMetricsGetsUpdated(): Unit = {
+    val props = TestUtils.createBrokerConfig(1, "localhost:1234")
+    val config = KafkaConfig.fromProps(props)
+
+    val mockBlockingSend: BlockingSend = createNiceMock(classOf[BlockingSend])
+
+    val log: Log = createNiceMock(classOf[Log])
+
+    val partition: Partition = createNiceMock(classOf[Partition])
+    expect(partition.localLogOrException).andReturn(log)
+    expect(partition.isReassigning).andReturn(true)
```

Comment:
Can we add the inverse test assertion too? Where we aren't reassigning and ensure `reassignmentBytesInPerSec` stays 0